src/setup/process.rs

```diff
@@ -0,0 +1,58 @@
+//! Set of convenience function to manage rippled process.
+
+use std::{
+    net::SocketAddr,
+    process::{Child, Command, Stdio},
+    time::Duration,
+};
+
+use tokio::io::AsyncWriteExt;
+
+use crate::{setup::config::NodeMetaData, wait_until};
+
+pub const CONNECTION_TIMEOUT: Duration = Duration::from_secs(2);
+
+/// Stops the child and collects its exit message.
+pub fn stop(mut child: Child) -> Option<String> {
```

Comment:
not a huge fan of returning a `String` here; ideally, this would be sth like
```
enum ReturnCode {
    Success,
    ErrorCode(i32),
}
```

Reply:
also, we should be returning a `Result<ReturnCode>`, not an `Option`, and not use `ok` in the meantime; we don't want to obscure any OS errors

Reply:
Done